wgsl/index.bs

```diff
@@ -8969,6 +8974,11 @@ Each is described in detail in subsequent sections.
       <td>input
       <td>u32
 
+  <tr><td>[=built-in values/clip_distances=]
+      <td>vertex
+      <td>output
+      <td>array&lt;f32, N&gt;
```

Comment:
From what I can tell, the API requires `N` to be 8. Is there a reason not to restrict  WGSL to the same value?

Reply:
I've added the restriction to WGSL. PTAL, thanks!